eng/pipelines/repoman.yml

```diff
@@ -132,3 +122,21 @@ jobs:
             -Tag '<!-- #comment-repoman-generate -->'
         env:
           GH_TOKEN: $(azuresdk-github-pat)
+
+      - task: PowerShell@2
+        displayName: Repoman Clean PR Branch(s)
+        condition: >-
+          and(
+            succeeded(),
+            eq(variables['Build.SourceBranch'], 'refs/heads/main')
+          )
+        inputs:
+          pwsh: true
+          targetType: filePath
+          filePath: ./eng/scripts/Invoke-RepomanClean.ps1
+          arguments: >-
+            -CommitId '$(Build.SourceVersion)'
+            -Repo "azure/azure-dev"
+            -RunnerTemp './tmp'
```

Comment:
note that we are passing in ./tmp and not .tmp so it's actually a local temp folder in the working directory. This is because the clean command clones ensuring the target temp folder is empty. On hosted linux agents, the build agent user may not have permissions to delete files in ./tmp. We provided [System.IO.Path]::GetTempPath() as a default, but elected to clone the target repo to a local temp folder.